docs/api/effector-solid/useEvent.md

```diff
@@ -0,0 +1,99 @@
+---
+id: useEvent
+title: useEvent
+---
+
+Bind event to current fork instance to use in dom event handlers. Used **only** with ssr, in application without forks `useEvent` will do nothing
```

Comment:
I wouldn't agree, that `useEvent` is useful only with ssr 🤔 
e.g. it also can be useful if one wants to test components with something like RTL

Also i suggest to use `scope` instead of `fork instance`

I suggest something like this:
```suggestion
Bind event to current [_scope_](../effector/Scope.md) to use in dom event handlers.
Only `effector-solid/scope` version works this way, `useEvent` of `effector-solid` is no-op and does not require `Provider` with scope.
```

Reply:
This was just a copypaste from docs of effector-react, might I change the wording there as well?